components/zcash_address/src/kind/unified/fvk.rs

```diff
@@ -94,51 +90,75 @@ impl SealedItem for Fvk {
 /// // We can obtain the pool-specific Full Viewing Keys for the UFVK in preference
 /// // order (the order in which wallets should prefer to use their corresponding
 /// // address receivers):
```

Comment:
This comment is no longer correct for `ufvk.items_as_parsed()`.

Reply:
:heavy_check_mark: